api/controllers/kubernetes/install/app.go

```diff
@@ -11,15 +11,15 @@ import (
 )
 
 func (c *InstallController) GetAppConfig(ctx context.Context) (kotsv1beta1.Config, error) {
-	if c.releaseData == nil || c.releaseData.AppConfig == nil {
+	if c.appConfigManager == nil {
```

Comment:
are these checks still needed? we dont check that a manager is nil anywhere else

Reply:
right now there's a limitation where v3 doesn't work without a config. i can create a story to remove that limitation as i see the controller returns an error if there's no config at the moment.

Reply:
Agree, if it's data we provide at instantiation time we should do it there and only once, no point in spreading this nil check logic around unless we absolutely need to.